plugins/inputs/nginx_vts/nginx_vts.go

```diff
@@ -0,0 +1,344 @@
+package nginx_vts
+
+import (
+	"bufio"
+	"encoding/json"
+	"fmt"
+	"net"
+	"net/http"
+	"net/url"
+	"strings"
+	"sync"
+	"time"
+
+	"github.com/influxdata/telegraf"
+	"github.com/influxdata/telegraf/internal"
+	"github.com/influxdata/telegraf/plugins/inputs"
+)
+
+type NginxVTS struct {
+	Urls []string
+
+	client *http.Client
+
+	ResponseTimeout internal.Duration
+}
+
+var sampleConfig = `
+  ## An array of ngx_http_status_module or status URI to gather stats.
+  urls = ["http://localhost/status"]
+
+  # HTTP response timeout (default: 5s)
```

Comment:
Use two pounds for config comments - (`##`) and we generally comment default settings out in the sampleconfig if you'd like to do that too.